internal/platform.go

```diff
@@ -1,11 +1,11 @@
-package link
+package internal
 
 import (
 	"fmt"
 	"runtime"
 )
 
-func platformPrefix(symbol string) string {
+func PlatformPrefix(symbol string) (string, bool) {
```

Comment:
Maybe `PlatformPrefix() string` with `""` meaning unknown prefix is more convenient?

Nit: can you add a doc comment?